lib/mix/test/test_helper.exs

```diff
@@ -269,6 +269,15 @@ Enum.each(fixtures, fn fixture ->
   File.cp_r!(source, dest)
 end)
 
+## Set up Gleam fixtures
+
```

Comment:
And we should probably skip gleam tests if gleam is not installed, right?

Reply:
Done. I guess I need to add gleam to CI (although I need to wait until my PR is merged and released).